frontend/packages/topology/integration-tests/features/topology/topology-editing-app-node.feature

```diff
@@ -50,3 +50,155 @@ Feature: Editing an application
               And user clicks on Save
              Then user is redirected to topology
               And user can see a toast notification of JAR file uploading with link to build logs
+
+
+        @regression @manual
+        Scenario: Edit form for deployment
+            Given user has created deployment workload
+              And user is at the Topology page
+             When user right clicks on the node to open context menu
+              And user selects option "Edit Deployment" from context menu
+             Then user will see option to switch between YAML view and Form view
+              And user will see Recreate Strategy type under Deployment Strategy selected by default
+              And user will see "Maximum number of unavailable Pods" and "Maximum number of surge Pods" sections for Strategy type RollingUpdate under Deployment Strategy
+              And user will see "Deploy image from an image stream tag" checkbox under Images with project, image stream and tag section associated with it
+              And user will see Image name section under Images with checkbox "Deploy image from an image stream tag" unchecked
+              And user will see "Show advanced image options" and "Show advanced container options" in case of image stream option
+              And user will see advanced option for "Pause rollouts" and "Scaling"
```

Comment:
When we are editing deployment workload in below scenarios, at that point of time, we are verifying these content so IMO, this scenario is required.

Reply:
This has been covered to check all the options present in the form